tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/AgentMalfunctionTests.cs

```diff
@@ -82,7 +82,13 @@ public async Task SubmitsTraces(AgentBehaviour behaviour, TestTransports transpo
 
         private async Task TestInstrumentation(MockTracerAgent agent, string metadataSchemaVersion)
         {
-            const int expectedSpanCount = 5;
+            int expectedSpanCount = 5;
```

Comment:
This seems to be expected as the snapshots that use this sample project only have 3 spans on Linux/OSX